packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,161 @@
+| `tfe_database_name`                          | no       | `"tfe"`                                      | TFE database name.                                                                                                                                                                                                                                                                  |
+| `tfe_database_parameters`                    | no       | `"sslmode=require"`                          | TFE database server parameters for the connection URI.                                                                                                                                                                                                                              |
+| `tfe_object_storage_type`                    | no       | `"s3"`                                       | Type of object storage to use. Must be one of s3, azure, or google.                                                                                                                                                                                                                 |
+| `tfe_run_pipeline_nomad_address`             | yes      | `""`                                         | The server address of Nomad where TFE is being deployed.                                                                                                                                                                                                                            |
+| `tfe_object_storage_s3_bucket`               | no       | `"tfe"`                                      | The bucket name of the S3 compatible object storage being used.                                                                                                                                                                                                                     |
+| `tfe_object_storage_s3_region`               | no       | `"us-west-2"`                                | S3 region.                                                                                                                                                                                                                                                                          |
+| `tfe_object_storage_s3_use_instance_profile` | no       | `"false"`                                    | Whether to use the instance profile for authentication.                                                                                                                                                                                                                             |
```

Comment:
```suggestion
| `tfe_object_storage_s3_use_instance_profile` | no       | `false`                                    | Whether to use the instance profile for authentication.                                                                                                                                                                                                                             |
```

Reply:
`variable "tfe_object_storage_s3_use_instance_profile" {
  description = "The instance profile setting for accessing the object storage bucket to be used by TFE"
  type        = string
  default     = "false"
}`

It is a string, TFE does not expect anything as a boolean, it expects them as a string.